translations/zh_CN/topics.po

```diff
@@ -7810,19 +7810,19 @@ msgid ""
 "``RuntimeError`` will be raised. In this case, the conflicting argument name"
 " needs to be changed to something that does not overlap with the "
 "automatically generated labels for the variadic arguments."
-msgstr ""
+msgstr "结果将是 ``Float`` node(示例中的值为 ``2 ``. Since in this example the arguments are not explicitly declared in the function signature, nor are their values passed with a keyword in the function invocation, AiiDA needs to come up with a different way to determine the labels to link the input nodes to the calculation. For variadic arguments, link labels are created from the variable argument declaration (`` *args``)，后面跟一个索引。因此，上例中的链接标签将是 ``args_0`` 、 ``args_1`` 和 ``args_2`` 。如果这些标签中的任何一个与位置参数或关键字参数的标签重叠，就会出现 ``RuntimeError`` 。在这种情况下，需要将有冲突的参数名称改为不与自动生成的变量参数标签重叠的名称。"
```

Comment:
```suggestion
msgstr "结果将是 ``Float`` node(示例中的值为 ``2``.  Since in this example the arguments are not explicitly declared in the function signature, nor are their values passed with a keyword in the function invocation, AiiDA needs to come up with a different way to determine the labels to link the input nodes to the calculation. For variadic arguments, link labels are created from the variable argument declaration (``*args``)，后面跟一个索引。因此，上例中的链接标签将是 ``args_0`` 、 ``args_1`` 和 ``args_2`` 。如果这些标签中的任何一个与位置参数或关键字参数的标签重叠，就会出现 ``RuntimeError`` 。在这种情况下，需要将有冲突的参数名称改为不与自动生成的变量参数标签重叠的名称。"
```